Source/Widgets/Geocoder/GeocoderViewModel.js

```diff
@@ -53,33 +66,183 @@ define([
         }
         //>>includeEnd('debug');
 
+        this._geocoderServices = options.geocoderServices;
+        if (!defined(options.geocoderServices)) {
+            this._geocoderServices = [
+                new LongLatGeocoderService(),
+                new BingMapsGeocoderService(),
+                new OpenStreetMapNominatimGeocoderService()
```

Comment:
Not sure if we want to include this one by default.  Keep `LongLat` and `Bing` though